core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -339,6 +339,10 @@ public ClientTransport get(CallOptions callOptions) {
     if (log.isLoggable(Level.INFO)) {
       log.log(Level.INFO, "[{0}] Created with target {1}", new Object[] {getLogId(), target});
     }
+
+    if (idleTimeoutMillis == IDLE_TIMEOUT_MILLIS_DISABLE) {
+      exitIdleMode();
```

Comment:
Let's stay in idle mode initially, even if it is disabled. That way constructing a Channel does no I/O.